gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/api/TestManager.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright � 2017, Battelle Memorial Institute All rights reserved.
```

Comment:
@tdtalbot  Did you pull in @Jeffrey-Simpson 's stuff?  So we have cross branches?  If so, merge it again and we can close one of the pull requests.